app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -8,7 +8,7 @@ export default function NoteTable({
 }) {
   return (
     <div className="w-full px-4">
-      <table className="nui-table nui-table--checkbox">
+      <table className="nui-table nui-table--checkbox  nui-table--actions">
```

Comment:
Please add className `nui-table--hover`.
This classsname adds hover effects to the table row.